### DISCUSSION TITLE
Why there is no "mn" field for visual-object?

### DISCUSSION THREAD

**Discussion by birhburh** [lottie/lottie-spec#99] [Q&A] [answered]
Sorry for bothering you
I'm just little confused
I'm trying to parse files generated by glaxanimate
Is there a reason why [lottie-spec visual-object](https://github.com/lottie/lottie-spec/blob/main/schema/helpers/visual-object.json)
differs from [lottie-docs visual-object](https://github.com/LottieFiles/lottie-docs/blob/main/docs/schema/helpers/visual-object.json)?
It was added to lottie-docs 2 years ago as I see. Do I better stick with lottie-docs schema for now?
Ah, and yes, files in this repo in tests use this field also

**Comment by mbasaglia** [comment, marked as answer]
`lottie-docs` aims at documenting everything you can find in a lottie. 
`lottie-spec` aims at providing a common foundation to ensure players and editors agree on how things work.

Version 1.0 of lottie-spec contains the minimum set of features common to most players, more stuff will be added to it as time goes on.

**Reply by birhburh** [reply]
Thanks!
